languages.toml

```diff
@@ -68,6 +68,7 @@ swipl = { command = "swipl", args = [ "-g", "use_module(library(lsp_server))", "
 taplo = { command = "taplo", args = ["lsp", "stdio"] }
 terraform-ls = { command = "terraform-ls", args = ["serve"] }
 texlab = { command = "texlab" }
+ucm = { command = "ncat", args = ["localhost", "5757"] } 
```

Comment:
Similar to gdscript this is kind of a hack and might be better off living in the wiki instead of the default config https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers#gdscript